### ISSUE TITLE
Only prints escape code on win 10

### ISSUE THREAD

**Issue by idelem** [charmbracelet/glow#127]
I typed `> .\glow.exe` in **powershell**. Instead of the README.md text, it output the following sequence:
```
?[1;38;5;228;48;5;63m?[0m?[1;38;5;228;48;5;63m?[0m  ?[1;38;5;228;48;5;63m ?[0m?[1;38;5;228;48;5;63mGlow?[0m?[1;38;5;228;48;5;63m ?[0m?[38;5;252m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[38;5;252m ?[0m?[0m
[... and a lot more of this]
```
To make sure it's not an issue with powershell, I tried again in **cmd.exe** and produced the same result. 
Looks like ANSI escape code is broken on win 10? Or is it just my machine.

**Comment by muesli** [comment]
I thought they were re-enabled in a later Windows 10 update. Are you running the latest version?

You can still manually activate it, though. See this link: https://stackoverflow.com/questions/51680709/colored-text-output-in-powershell-console-using-ansi-vt100-codes

I'll try to enable this for glow via an API call, but I was hoping to get away without it 😅

**Comment by muesli** [comment]
I just released `glow v0.2.0` which should enable ANSI support on Windows 10 without manual intervention. I'll close this ticket, but should you still run into problems, please let us know!

**Comment by idelem** [comment]
> I just released `glow v0.2.0` which should enable ANSI support on Windows 10 without manual intervention. I'll close this ticket, but should you still run into problems, please let us know!

Thanks! I'll give it a try
